setup.py

```diff
@@ -26,14 +27,16 @@ def _get_description():
     long_description_content_type="text/markdown",
     entry_points={"console_scripts": ["eco_parser = eco_parser.__main__:main"]},
     install_requires=["lxml", "requests"],
```

Comment:
I wasn't sure about pinning these libraries to a specific version so I left them unpinned.